### PR TITLE
Dnbr sentinel 2 nrt

### PR DESCRIPTION
**Pull request by james-blitzm**
Implements 3-band burnscar process (dNBR, BSI and NDVI).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -129,3 +169,21 @@ wo-deadletter:
...
+index_nci:
+	docker-compose exec alchemist \
+		bash -c "\
+			find /code/examples/ga_ls8c_ard_3/ -name "*.odc-metadata.yaml" | \
+			xargs datacube dataset add \
+			--ignore-lineage --confirm-ignore-lineage --product=ga_ls8c_ard_3 \
+		"
+
+process_nci:
+	echo "Fake"
```
Comment (alexgleith) [inline Makefile:189]: Sorry, this is my bad. Can we please remove this?

File: examples/c3_config_dnbr_3band.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    static_stretch: [-1, 1]
+  explorer_url: https://explorer.dev.dea.ga.gov.au
+  write_data_settings:
+    overview_resampling: average
+  reference_source_dataset: True
+  write_stac: True
+  inherit_geometry: True
+
+  metadata:
+    product_family: dnbr
```
Comment (alexgleith) [inline examples/c3_config_dnbr_3band.yaml:34]: Let's change this
```suggestion
    product_family: bs
```

File: examples/c3_config_dnbr_3band.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    producer: ga.gov.au
+    dataset_version: 0.0.2
+    collection_number: 3
+    naming_conventions: dea_c3
```
Comment (alexgleith) [inline examples/c3_config_dnbr_3band.yaml:38]: Can you please make this the same as the one Matthew uses for WO on S2.

**Review by alexgleith — CHANGES_REQUESTED** [review]

File: examples/c3_config_dnbr_3band.yaml (hunk 3 above)
Comment (james-blitzm) [reply examples/c3_config_dnbr_3band.yaml:38]: Change applied.

File: examples/c3_config_dnbr_3band.yaml (hunk 2 above)
Comment (james-blitzm) [reply examples/c3_config_dnbr_3band.yaml]: Change applied.

File: Makefile (hunk 1 above)
Comment (james-blitzm) [reply Makefile]: Removed.

**Review by alexgleith — APPROVED** [review]

**Pull request merged**